### DISCUSSION TITLE
Visual Editor

### DISCUSSION THREAD

**Discussion by impguard** [matteobruni/tsparticles#2755] [Q&A] [answered]
Is there a visual editor for tsparticles? I have a few designers who aren't familiar with coding/json etc. and would prefer a form editor similar to https://pixijs.io/pixi-particles-editor/#pixieDust.

Where they can upload some pictures, mess around with configuration options, and then export the config to me. The demo page for tsparticles seems close but you can't seem to really edit things too much vs modify a preset.

**Comment by matteobruni** [comment, marked as answer]
On the website there are two editors, the custom one in the homepage and one using JSON editor in the demos page. I have nothing more for now. The custom one is in this repository, if you need to change things you can grab the source or submit a PR.

**Reply by impguard** [reply]
makes sense. I think both don't really support actually fully editing the JSON, unless I'm just clueless about the editor in the demos page. Thanks for the update though.
